info/diktat-kotlin-coding-style-guide-en.md

```diff
@@ -1045,35 +1045,35 @@ fun baz() {
 
 ### <a name="s3.7"></a> Recommendation 3.7: Usage of whitespace for code separation
 
-  1. Any keywords (like 'if', 'when', 'for', e.t.c) should be separated with a single whitespace from the opening parenthesis.
-     Only exceptions are: 'super' and 'constructor' keywords. They should not be separated from a parenthesis.
+  1. Any keywords (like 'if', 'when', 'for', e.t.c) should be separated from with a single whitespace from the opening parenthesis.
+     Only exceptions is 'constructor' keyword. It should not be separated from a parenthesis.
 
-  2. Separate any keywords (such as `else` or `catch`, etc.) from the opening brace ('{') with a single whitespace.
+  2. Separate any keywords (such as `else` or `try`, etc.) from the opening brace ('{') with a single whitespace.
   
   3. Use single whitespace before any opening brace (`{`).
-     Only exception is passing a lambda as an argument inside parenthesis:
+     Only exception is passing of a lambda as a parameter inside parenthesis:
      ```kotlin
          private fun foo(a: (Int) -> Int, b: Int) {}
          foo({x: Int -> x}, 5) // no space before '{'
      ```
 
   4. Single whitespace should be placed on both sides of binary operators. This also applies to operator-like symbols, for example: 
 
-     - Colon in generic structures with 'where' keyword： `where T : Type`
-     - Arrow in lambdas： `(str: String) -> str.length()`
+     - Colon in generic structures with 'where' keyword: `where T : Type`
+     - Arrow in lambdas: `(str: String) -> str.length()`
 
   **Exceptions：**
 
   - Two colons (`::`) are written without spaces: `Object::toString`
-  - Dot separator (`.`) that stays on the same line with an object name: `object.toString()`
+  - Dot separator (`.`) that stays on the same line with an object name `object.toString()`
 
   5. Spaces should be used after ',',':',';' (except cases when those symbols are in the end of line). There should be no whitespaces in the end of line.
 
   6. There should be *only one space* between identifier and it's type： `list: List<String>`
   
   7. When using '[]' operator (get/set) there should be *no* spaces between identifier and '[': `someList[0]`
   
-  8. There should be no space between a method name and a parenthesis: `foo() {}`
+  8. There should be no space between a method or constructor name and a parenthesis (at call site? at declaration site? both?): `foo() {}`
```

Comment:
if it is not clear - please also add a note : 
"Note that this subrule is related only to spaces, whitespace rules are described in ###. So this rule does not prohibit the following code: fun foo(
a: String"